cmake/FindSphinx.cmake

```diff
@@ -1,7 +1,48 @@
-find_program(SPHINXBUILD
-  sphinx-build
+# Distributed under the OSI-approved Apache 2.0. See the LICENSE file in
+# FoundationDB source code
+
+#[=======================================================================[.rst:
+FindSphinx
+-------
+
+Find Sphinx, the Python documentation generator
+
+Sphinx_ROOT variable can be used for HINTS for different version of Sphinx.
+
+Result variables
+^^^^^^^^^^^^^^^^
```

Comment:
Is this something of cmake convention?

Reply:
This is the documentation format used in other `FindXXXX.cmake`. I am not sure if it is standard but it seems used by a lot of standard modules.